OpenRA.Mods.Common/Projectiles/Railgun.cs

```diff
@@ -108,6 +110,11 @@ public class Railgun : IProjectile
 
 		int ticks;
 		bool animationComplete;
+
+		[Sync]
+		WPos source;
```

Comment:
We don't appear to be using this anywhere?

Reply:
You're right - it's only used in the constructor.